### DISCUSSION TITLE
Need help with this error code - Items not dropping upon defeating any enemy/boss

### DISCUSSION THREAD

**Discussion by LightningHusky** [Pryaxis/TShock#2623] [Q&A]
Hi Tshock Community,

I'd like to ask if somebody could help me with this error code:
[2022-05-02_17-17-27.log](https://github.com/Pryaxis/TShock/files/8603990/2022-05-02_17-17-27.log)
Any help would be appreciated thank you =)

Also can somebody update the Antispam Plugin? I'd be doing it, but I have no idea how to update/code plugins  =(

**Comment by punchready** [comment]
I cannot reproduce this in a fresh download of the latest tShock build using the exact same commands as you, do you have any special plugins or modifications in place?

The exception thrown is also very weird, the only variable that can be null in this case should never be null because it is used in an earlier call, at least based on the current code.

**Reply by LightningHusky** [reply]
I just launched up the regular Tshock Server no Plugins nothing.
I use Nitrado as my Serverhost
